.github/workflows/license_finder.yml

```diff
@@ -36,7 +36,8 @@ jobs:
       - name: Generate requirements.txt (exclude dev dependencies)
         run: |
           poetry export -f requirements.txt --without-hashes > requirements.txt
+          pip install -r requirements.txt
 
       - name: Run license finder
         run: |
-          poetry run license_finder --python-version=3
+          /opt/hostedtoolcache/Ruby/*/x64/lib/ruby/gems/*/gems/license_finder-*/bin/license_finder_pip.py requirements.txt
```

Comment:
I think we need to invoke this via `python3`. It looks like in the failed jobs it's `python3 /path/to/license_finder_pip.py /path/to/requirements.txt`, and in successful runs in other SDKs the output is different from here (it should say `All dependencies are approved for use`).

Reply:
Let me try that!

Reply:
You're absolutely right about the expected output! Unfortunately using python3 was not the method. It was the script itself, so I made that fix. Thanks for catching that!

Reply:
I realized that we still need this command to check license finder and fail the workflow if needed. I checked, and for some reason `license_finder` returns a exit status of 0 even though it's failing.